motr/ut/idx_dix.c

```diff
@@ -399,12 +416,24 @@ static void ut_dix_namei_ops(bool dist)
 
 static void ut_dix_namei_ops_dist(void)
 {
-	ut_dix_namei_ops(true);
+	ut_dix_namei_ops(true, 0);
+}
+
+static void ut_dix_namei_ops_dist_oi_flags_skip_layout(void)
```

Comment:
Rename ut_dix_namei_ops_dist_oi_flags_skip_layout() to  ut_dix_namei_ops_skip_layout()
And do the same for other corresponding API's .

Reply:
Done.